Integrations/integration-ArcherRSA.yml

```diff
@@ -2944,10 +2947,15 @@ script:
         var incidents = createIncidentsArr(results, dataArgs.applicationId);
 
         if (incidents.length > 0){
+            shouldAddOffset = true;
             lastTime = incidents[incidents.length-1].occurred;
-        }
 
-        setLastRun({ lastTime: lastTime + offset});
+            if (params.useEuropeanTime) {
```

Comment:
why do we need this condition? seems to be always true

Reply:
Because we're not sure yet if the createIncidentsArr function works perfectly for American time.
Sent a version without this condition to be tested by KPMG, if it works for them I'll update this version